include/script_api/macros.h

```diff
@@ -125,10 +126,18 @@
 ///         Bytecode argv[argc];
 ///     }
 /// This macro expands to the given opcode and argv, with argc calculated automatically.
+
+#ifndef PERMUTER
 #define EVT_CMD(opcode, argv...) \
     opcode, \
     (sizeof((Bytecode[]){argv})/sizeof(Bytecode)), \
     ##argv
+#else
+#define EVT_CMD(opcode, argv...) \
+    opcode, \
+    0, \
```

Comment:
I'd make in `0xDEADBEEF` to be clear that this is dummy info because pycparser is noncompliant

Reply:
I'd rather just leave a comment - putting a random constant in like that feels odd